lib/messager.rb

```diff
@@ -0,0 +1,21 @@
+require 'dotenv'
+require 'twilio-ruby'
+
+class Messager
+
+  attr_reader :content_manager
+
+  def initialize(content_manager)
+    Dotenv.load
+    @content_manager = content_manager
+  end
+
+  def send_text(order_cost)
+    Twilio::REST::Client.new(ENV['TWILIO_SID'], ENV['TWILIO_AUTH_TOKEN']).
```

Comment:
set this as a default client so that you can inject a double in its place:

```ruby
def initialize(content_manager, twilio_client)
  @twilio_client = twilio_client || Twilio::REST::Client.new(etc)
  @content_manager = content_manager
end
```